### DISCUSSION TITLE
Plugin ID mismatch

### DISCUSSION THREAD

**Discussion by xrff** [oxdc/obsidian-vertical-tabs#58] [Q&A] [answered]
<img width="181" alt="image" src="https://github.com/user-attachments/assets/ef89abe9-b5e5-4284-8c80-be6cbdb86f6e">

can not install this plugin. obsidian version 1.4.16

**Comment by oxdc** [comment, marked as answer]
_Vertical Tabs_ has not been thoroughly tested in old versions of Obsidian. Installation is, therefore, disabled to prevent inconsistent performance. If you have no particular reason for staying in this version, please consider **downloading the latest installer of Obsidian** to update.

**Comment by xrff** [comment]
> _Vertical Tabs_ has not been thoroughly tested in old versions of Obsidian. Installation is, therefore, disabled to prevent inconsistent performance. If you have no particular reason for staying in this version, please consider **downloading the latest installer of Obsidian** to update.

ok, resolved, thanks !
